docs/docs/community/release-process.md

```diff
@@ -0,0 +1,75 @@
+---
+id: release-process
+title: The Nebari release process
+description: The Nebari release process and workflow
+---
+
+# The Nebari release process
+
+This page describes the high-level details of cutting a new Nebari release.
+
+|                   |                                                                                                                                                                                                  |
+| :---------------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|      Cadence      |                                                                                             Monthly                                                                                              |
```

Comment:
Monthly - on the fourth week of the month

Reply:
I updated this with the `3rd week of the month`. This gives us a little buffer ;)